src/Microsoft.DotNet.Interactive.Tests/Server/SerializationTests.cs

```diff
@@ -315,6 +315,7 @@ IEnumerable<KernelEvent> events()
                     new ResolvedPackageReference(
                         packageName: "ThePackage",
                         packageVersion: "1.2.3",
+                        false,
```

Comment:
Kevin, do you still need this?

Reply:
I still haven't checked ...

Reply:
Yes I do:   here:  https://github.com/dotnet/interactive/pull/1247/files#diff-7c24de179534b7b7914d7b376592645b4fca966c1a4d87d2ea2bda1a6bcbbe84R123